src/deepsparse/loggers/metric_functions/helpers/config_generation.py

```diff
@@ -0,0 +1,34 @@
+__all__ = ["data_logging_config_from_predefined"]
+
+from typing import Dict, List, Optional, Union, Any
+
+from deepsparse.loggers.build_logger import default_logger
+
+
+def data_logging_config_from_predefined(
+    group_names: Union[str, List[str]],
+    loggers: Dict[str, Optional[Dict[str, Any]]] = default_logger(),
```

Comment:
been leaving this comment a lot - we cannot use mutable types as function defaults in python as they can have unintended side effects. read more here: https://florimond.dev/en/posts/2018/08/python-mutable-defaults-are-the-source-of-all-evil/